<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1005</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Nama Usaha</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Komoditas</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>LuasSK</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtNamaUsaha">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>90</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtKomoditas">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>140</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtLuasSk">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>200</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSimpan">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>270</y>
     <width>90</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Simpan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUbah">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>270</y>
     <width>90</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Ubah</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHapus">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>270</y>
     <width>90</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Hapus</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBersih">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>270</y>
     <width>90</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Bersih</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableIup">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>340</y>
     <width>751</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>id_iup</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>nama usaha</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>komoditas</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Luas sk</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>desa</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>sungai besar</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>120</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Desa</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbDesa">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>120</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbSungaiBesar">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>180</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>180</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Sungai Besar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
